contracts/cw3-multisig/src/state.rs

```diff
@@ -107,17 +107,17 @@ impl Proposal {
         self.status = self.current_status(block);
     }
 
-    // returns true iff this proposal is sure to pass (even before expiration if no future
-    // sequence of possible votes can cause it to fail)
-    pub fn is_passed(&self, block: &BlockInfo) -> bool {
+    /// Helper function to check if a certain vote count has reached threshold.
+    /// Only called from is_rejected and is_passed for no and yes votes
+    fn does_vote_count_reach_threshold(&self, vote_count: u64, block: &BlockInfo) -> bool {
```

Comment:
Good generalisation. A little unclear what this does, but I think with the comment above, it's OK.

Reply:
I'm going to look into making it clearer via making the comments better today.

Reply:
Have attempted to improve the comments in my latest commits, feedback welcome.